Userland/Shell/Tests/valid.sh

```diff
@@ -65,6 +65,7 @@ word_count=(() | wc -w)
 if not test "$(echo well hello friends $word_count)" -eq 3  { fail variable containing pipeline }
 
 # Globs
+rm -fr /tmp/sh-test 2> /dev/null
```

Comment:
> Agni: who the fuck types `rm -fr`

nice

Reply:
Ey man, didn't want to have the rm -fr at the bottom of the file be inconsistent